srw/srw.cpp

```diff
@@ -458,7 +521,8 @@ class srw {
 
 				it->second->update(event, sph);
 
-				dnet_shared_upstream_t upstream(std::make_shared<dnet_upstream_t>(m_node, st, cmd, event, (uint64_t)sph->flags));
+				dnet_shared_upstream_t upstream(std::make_shared<dnet_upstream_t>( m_node, st, cmd, sph,
```

Comment:
Indent is broken at `std::make_shared<dnet_upstream_t>( m_node` - a space after `( `